modules/compliance-operator-rosa-installation.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-management/compliance-operator-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-compliance-operator-rosa_{context}"]
+= Installing the Compliance Operator on ROSA hosted control planes (HCP)
+
+As of the Compliance Operator 1.5.0 release, the operator is tested against Red Hat
+OpenShift Service on AWS (ROSA) using hosted control plane topologies (HCP).
```

Comment:
* Capitalize "Operator" per [repo term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#operator).
* There's a [common-attribute](https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc?plain=1#L164) available for the full ROSA name.
```suggestion
As of the Compliance Operator 1.5.0 release, the Operator is tested against {product-rosa} using hosted control plane topologies (HCP).
```
There are also some for HCP, if you like:
```
// Hosted control planes related attributes
:hcp-capital: Hosted control planes
:hcp: hosted control planes
```